src/devices/src/virtio/vsock/event_handler.rs

```diff
@@ -547,4 +569,70 @@ mod tests {
             GAP_SIZE as u32 + 100,
         );
     }
+
+    #[test]
+    fn test_event_handler() {
+        let mut event_manager = EventManager::new().unwrap();
+        let test_ctx = TestContext::new();
+        let EventHandlerContext {
+            device,
+            guest_rxvq,
+            guest_txvq,
+            ..
+        } = test_ctx.create_event_handler_context();
+
+        let vsock = Arc::new(Mutex::new(device));
+        event_manager.add_subscriber(vsock.clone()).unwrap();
+
+        // Push a queue event
+        // - the driver has something to send (there's data in the TX queue); and
+        // - the backend has no pending RX data.
```

Comment:
This comment says that the backend has no pending RX data, but in the code block below, you do:
`device.backend.set_pending_rx(true)`.

Reply:
You're right, I had updated the test but forgot to update the comment.
Fixed.